content/src/quick-action-tasks/electrical-contracting-business-permit-update.md

```diff
@@ -1,49 +1,56 @@
 ---
 filename: electrical-contracting-business-permit-update
-name: " Update Your Electrical Contracting Business Permit"
+name: "Update Your Electrical Contracting Business Permit"
 industryIds:
   - electrical-contractor
 sectorIds: []
 icon: update-purple.svg
 urlSlug: electrical-contracting-business-permit-update
 callToActionText: Update My Electrical Contracting Business Permit
 callToActionLink: https://www.njconsumeraffairs.gov/elec/Pages/applications.aspx
----
-You need to update your electrical contracting business permit if you are making changes to:
+summaryDescriptionMd: >-
+  You need to update your electrical contracting business permit if you are making changes to:
+
+
+  - Your business name
+
+  - Your trade status
 
-* Your business name
-* Your trade status
-* The licensee of your business, if your business permit is active 
+  - The licensee of your business, if your business permit is active
 
-:::infoAlert 
- It is recommended to apply at least ten days **before** the next scheduled monthly meeting of the Board of Examiners of Electrical Contractors. Dates are posted on the [Meetings](https://www.njconsumeraffairs.gov/elec/Pages/meetings.aspx) page.
-:::
+
+  :::infoAlert
+
+  It is recommended to apply at least ten days **before** the next scheduled monthly meeting of the Board of Examiners of Electrical Contractors. Dates are posted on the [Meetings](https://www.njconsumeraffairs.gov/elec/Pages/meetings.aspx) page.
+
+  :::
+---
 
 ### Application Requirements
```

Comment:
[pebble] Should this be a third-level heading as denoted here (`###`) or second-level heading (`##`)? If it's the first and only heading, I noticed our pattern is second-level.

Reply:
Good catch.